File: bench/fixtures/multilang_game/client/lua/arena_reward.lua
-- arena_reward client handler; config id 100126
local ArenaReward = {}

function ArenaReward.ClaimArenaReward(player)
  local cfg_id = 100126
  local key = "arena_reward"
  -- LUA_ONLY_CLUE_MoonlitArenaChest
  return net.send("ClaimArenaReward", cfg_id, key)
end

return ArenaReward

File: bench/fixtures/multilang_game/client/csharp/ArenaReward.cs
namespace Game.Client.Arena
{
    // ClaimArenaReward / arena_reward / 100126
    public static class ArenaReward
    {
        public const int ConfigId = 100126;
        public const string Key = "arena_reward";

        public static void ClaimArenaReward(long playerId)
        {
            Net.Send("ClaimArenaReward", ConfigId, Key);
        }
    }
}
